.github/workflows/pullrequest.yml

```diff
@@ -5,7 +5,8 @@ on:
       - master
 
 env:
-  EXTENSIONS: "c h cpp py go sh"
+  EXTENSIONS: "c h cpp py go sh mk spec service"
+  FILES: "*Makefile"
```

Comment:
Maybe rename to "EXTENSIONLESS_FILES"?

Reply:
Does this syntax with *asterisk work when Makefile is not in parent directory?

Reply:
Yes, asterisk works. I've put it there because filenames we get from github are full paths, so asterisk is needed to cover all Makefiles in all directories.

And about EXTENSIONLESS_FILES... it is not entirely true. Well, it is in this case, but if you take a look at the same PR in OCL (https://github.com/Open-CAS/open-cas-linux/pull/1203) you can see that there are more files included in this variable, among which there are 'configure.d/*' - this directory contains files with .conf extension. We usually don't want config files to contain copyright header, but those files are an exception.
So I would say that in FILES we put all files that doesn't have an extension or they have but it's different from the one that we want to check every time... or any other reason. ;) So instead of FILES_WITHOUT_OR_WITH_DIFFERENT_EXTENSION I decided to name this just FILES. :D